onionconfig/metaconfig.py

```diff
@@ -33,15 +35,15 @@ class BaseDimension(object):
     priority_class = 0
     valueset = []
     
-    def __init__(self, name, label=name, description=None, priority_class=0, valueset = list()):
+    def __init__(self, name, label=name, description=None, priority_class=0, valueset=list()):
```

Comment:
Having `list()` here is a bug. This instance will be shared across all instances.

Reply:
Fixed this

Reply:
Unfortunately, `valueset` as a list shared across all `BaseDimension` instances is the desired/expected behavior where the library is used downstream. Added this as a known issue but leaving bug unfixed for now.